### ISSUE TITLE
--output=table and --output=html sort fields in opposite order

### ISSUE THREAD

**Issue by folknor** [BePo65/license-report#59]
There's an issue where `license-report --output=table --fields name --fields licenseType --fields link --fields author` creates a table with the fields in the order specified in the cmdline, but `--output=html` reverses the order of the fields. So it looks like this:

![reversed](https://user-images.githubusercontent.com/233791/120883891-5f238e80-c5cf-11eb-9d64-df95cfbe02c2.jpg)

Thank you!

_Originally posted by @folknor in https://github.com/ironSource/license-report/issues/51#issuecomment-855198612_

**Comment by BePo65** [comment]
Thank you for opening this issue.
Are you sure that `--output=table` will also reverse the column order? At least on my development system I cannot reproduce tis effect.
For `output=html` I prepared a pr solving this issue - not yet pushed as a pr. If you want to try it out, you will find it as branch 'bepo65/issue59' in this project.

**Comment by folknor** [comment]
> Are you sure that `--output=table` will also reverse the column order? At least on my development system I cannot reproduce tis effect.

I didn't say that `--output=table` reverses the column order. I said it orders them like the CLI arguments, and that `--output=html` reverses it.

The issue title might be a bit ambigous, so perhaps that's where you got that impression.

I can test it after the 'pu/nightly' branch is merged.

Thank you!

**Comment by davidseek** [comment]
Faced the same problem, but I then just reversed the order in my config file 🤷🏻‍♂️

Before: 
```json
{
    "fields": [
        "name",
        "author",
        "link",
        "licenseType",
        "installedVersion"]
}
```

After: 

```json
{
    "fields": [
        "installedVersion",
        "licenseType",
        "link",
        "author",
        "name"
    ]
}
```
Is not perfect, but "fixes" my problem. So in a way, thank you for your issue.

[Result](https://mdsitelicenses.davidseek.md2site.com/)

**Comment by BePo65** [comment]
Fixed in next release based on pr #65.
